azure/v15.0.0/release.yaml

```diff
@@ -52,7 +52,7 @@ spec:
   - name: etcd
     version: 3.4.14
   date: "2021-06-14T11:40:38Z"
-  state: active
+  state: deprecated
```

Comment:
Not sure about this. 15.1.0 is on a different minor. I would leave this as `active` TBH

Reply:
all right 👍

Reply:
fixed